apps/site/lib/site_web/controllers/customer_support_controller.ex

```diff
@@ -140,8 +140,7 @@ defmodule SiteWeb.CustomerSupportController do
 
   @spec do_submit(Plug.Conn.t(), map) :: Plug.Conn.t()
   defp do_submit(%Plug.Conn{assigns: %{ip_address: {:ok, ip}}} = conn, data) do
-    rate_limit_interval =
-      if Application.get_env(:feedback, :rate_limit, true), do: 60_000, else: 1_000
+    rate_limit_interval = Application.get_env(:feedback, :rate_limit, 60_000)
```

Comment:
Is this to avoid timeouts when submitting the form (in real life and in the tests)?

Reply:
It leaves the default rate limit at 60s (so if someone tries to submit the form more than once a minute, they get stopped). Using the env variable lets me override that, which I do for tests (where many tests running very quickly sometimes in parallel will very easily hit the rate limit! so in the config file I set this value to 1s instead)

Reply:
@amaisano Hm yeah, that was pretty out of scope of me 🤔